magus_helpers/treeutils.py

```diff
@@ -96,6 +96,12 @@ def decomposeGuideTree(subsetsDir, sequencesPath, guideTreePath, maxSubsetSize,
     guideTree = dendropy.Tree.get(path=guideTreePath, schema="newick", preserve_underscores=True)
     guideTree.collapse_basal_bifurcation()
```

Comment:
This is nitpick by this point, but the order here: first collapse basal bifurcation, then resolve polytomies, seems to always recreate a basal bifurcation, which can be undesirable. I think first resolve polytomies, then resolve basal bifurcation might be a bit safer. Would this be correct? I am not sure and I just think this should be discussed before merging.

Reply:
I agree that should be the case. Is it possible to change the pull request? Maybe I should create a different one.

Reply:
Yep thanks for changing it... I am gonna leave the rest for Vlad (unless Vlad says he's busy)